e2_trainer.py

```diff
@@ -0,0 +1,86 @@
+import torch
+from torch.utils.data import DataLoader
+from tqdm import tqdm
+from torch.nn import functional as F
+from accelerate import Accelerator
+from e2_collate import collate_fn
+import os
+import logging
+from utils.compute_mel import TorchMelSpectrogram
```

Comment:
should be `from e2_tts_pytorch.utils.compute_mel`

Reply:
so should i move utils into e2_tts_pytorch/e2_tts_pytorch ?

Reply:
@manmay-nakhashi yup, you should move everything under that folder except for `train_e2.py`